Description: Patch README for Debian
 Remove installation instructions, and references to images in
 GitHub's CDN -- we expect the README to be viewed in plain text.
Author: Sean Whitton <spwhitton@spwhitton.name>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/README.md
+++ b/README.md
@@ -1,9 +1,3 @@
-[![Build](https://travis-ci.org/gorhill/uBlock.svg?branch=master)](https://travis-ci.org/gorhill/uBlock)
-[![Crowdin](https://d322cqt584bo4o.cloudfront.net/ublock/localized.png)](https://crowdin.com/project/ublock)
-[![License](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://github.com/gorhill/uBlock/blob/master/LICENSE.txt)
-
-***
-
 ##### BEWARE! uBlock Origin is COMPLETELY UNRELATED to the web site ublock.org**
 
 The donations sought by the [individual](https://github.com/chrisaljoudi/) behind `ublock.org` (_"to keeps uBlock development possible"_, [a misrepresentation](https://en.wikipedia.org/wiki/UBlock_Origin#uBlock_.28ublock.org.29)) are _not_ benefiting any of those who contributed most to create uBlock Origin ([developers](https://github.com/gorhill/uBlock/graphs/contributors), [translators](https://crowdin.com/project/ublock), and all those who put efforts in opening detailed issues). For the differences between uBlock Origin and uBlock, see the unbiased [Wikipedia article](https://en.wikipedia.org/wiki/UBlock_Origin).
@@ -15,11 +9,6 @@ uBlock Origin is **NOT** an "ad blocker"
 *** 
 
 <h1 align="center">
-<sub>
-<img  src="https://raw.githubusercontent.com/gorhill/uBlock/master/doc/img/icon38@2x.png"
-      height="38"
-      width="38">
-</sub>
 uBlock Origin
 </h1>
 <p align="center">
@@ -28,7 +17,6 @@ uBlock Origin
 </sup>
 <br>
 <sup> <!-- Languages -->
-      <img src="https://raw.githubusercontent.com/gorhill/uBlock/master/doc/img/languageicon-36.png" width="18" height="18">
       <sup>
             English,
             <a href="https://github.com/fang5566/uBlock/blob/master/README.md#ublock-origin">Chinese (中文)</a>,
@@ -38,22 +26,10 @@ uBlock Origin
 </sup>
 </p>
 
-
 **An efficient blocker add-on for various browsers. Fast, potent, and lean.**
 
 * [Documentation](#documentation)
 * [Purpose & General Info](#philosophy)
-* [Performance and Efficiency](#performance)
-  * [Memory](#memory)
-  * [CPU](#cpu)
-  * [Blocking](#blocking)
-  * [Quick tests](#quick-tests)
-* [Installation](#installation)
-  * [Chromium](#chromium)
-  * [Firefox](#firefox--firefox-for-android)
-  * [Microsoft Edge](#microsoft-edge)
-  - [Safari (macOS)](#safari-macos)
-* [Release History](#release-history)
 * [Privacy policy](https://github.com/gorhill/uBlock/wiki/Privacy-policy)
 * [Wiki](https://github.com/gorhill/uBlock/wiki)
 
@@ -78,87 +54,6 @@ _EasyList_, _Peter Lowe's Adservers_, _E
 
 Once you install uBlock₀, you may easily un-select any of the pre-selected filter lists if you think uBlock₀ blocks too much. For reference, Adblock Plus installs with only _EasyList_ enabled by default.
 
-## Performance
-
-#### Memory
-
-<div align="center">
-On average, uBlock Origin does make your browser run leaner. <sup>[1]</sup><br><br>
-
-Chromium<br>
-<img src="https://cloud.githubusercontent.com/assets/585534/10074141/15f04128-629c-11e5-9155-177fd4909083.png" /><br><br>
-
-Firefox<br>
-<img src="https://cloud.githubusercontent.com/assets/585534/10074130/0577118c-629c-11e5-9902-bf367c6a96c3.png" /><br><br>
-
-</div>
-
-<sup>[1] Details of the benchmark available at <a href="https://github.com/gorhill/uBlock/wiki/Firefox-version:-benchmarking-memory-footprint">Firefox version: benchmarking memory footprint</a>.</sup><br>
-
-#### CPU
-
-<p align="center">
-uBlock Origin is also easy on the CPU<br>
-<img src="https://raw.githubusercontent.com/gorhill/uBlock/master/doc/benchmarks/cpu-usage-overall-chart-20141226.png" /><br>
-<sup>Details of the benchmark available in <a href="https://github.com/gorhill/uBlock/blob/master/doc/benchmarks/cpu-usage-overall-20141226.ods">this LibreOffice spreadsheet</a>.</sup>
-</p>
-
-#### Blocking
-
-<p align="center">
-Being lean and efficient doesn't mean blocking less<br>
-<img src="https://raw.githubusercontent.com/gorhill/uBlock/master/doc/benchmarks/privex-201502-16.png" /><br>
-<sup>For details of benchmark, see 
-<a href="https://github.com/gorhill/uBlock/wiki/uBlock-and-others%3A-Blocking-ads%2C-trackers%2C-malwares">uBlock₀ and others: Blocking ads, trackers, malwares</a>.
-</p>
-
-## Installation
-
-Feel free to read [about the extension's required permissions](https://github.com/gorhill/uBlock/wiki/About-the-required-permissions).
-
-#### Chromium
-
-You can install the latest version [manually](https://github.com/gorhill/uBlock/tree/master/dist#install), from the [Chrome Store](https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm), or from the [Opera store](https://addons.opera.com/en-gb/extensions/details/ublock/).
-
-It is expected that uBlock Origin is compatible with any Chromium-based browsers.
-
-**Important:** Chromium-based browsers do not relay [websocket connections](https://en.wikipedia.org/wiki/WebSocket) to the extension API. This means websites can use websocket connections to bypass uBO (or any other blocker). This can be remediated by installing uBO's companion extension [uBO-Extra](https://github.com/gorhill/uBO-Extra).
-
-#### Firefox / Firefox for Android
-
-[Firefox Add-ons web site](https://addons.mozilla.org/addon/ublock-origin/). There is also a development version if you want to test uBlock Origin with the latest changes: see [_uBlock Origin Version History_](https://addons.mozilla.org/addon/ublock-origin/versions/beta)
-
-uBlock Origin is compatible with [SeaMonkey](http://www.seamonkey-project.org/), [Pale Moon](https://www.palemoon.org/), and possibly other browsers based on Firefox.
-
-The Firefox version of uBlock Origin has [an extra feature](https://github.com/gorhill/uBlock/wiki/Inline-script-tag-filtering) currently not yet available on Chromium-based browsers -- which feature is of great help to foil attempts by many web sites to circumvent blockers.
-
-Also of interest: [Deploying uBlock Origin for Firefox with CCK2 and Group Policy](http://decentsecurity.com/ublock-for-firefox-deployment/).
-
-Thanks to Debian contributor [Sean Whitton](https://wiki.debian.org/SeanWhitton), users of Debian 9 or later or Ubuntu 16.04 or later may simply
-`apt-get install xul-ext-ublock-origin`.
-
-#### Microsoft Edge
-
-Developer: [@nikrolls](https://github.com/nikrolls).
-
-Stable version available in [Microsoft Store](https://www.microsoft.com/store/p/app/9nblggh444l4).
-
-Development version available at <https://github.com/nikrolls/uBlock-Edge#edge>.
-
-#### Safari (macOS)
-
-Developer: [@el1t](https://github.com/el1t).
-
-Development version available at <https://github.com/el1t/uBlock-Safari#ublock-originfor-safari>.
-
-#### Note for all browsers
-
-To benefit from uBlock Origin's higher efficiency, it's advised that you don't use other inefficient blockers at the same time (such as AdBlock or Adblock Plus). uBlock₀ will do [as well or better](#blocking) than most popular ad blockers.
-
-## Release History
-
-See the [releases pages](https://github.com/gorhill/uBlock/releases) for a history of releases and highlights for each release.
-
 ## About
 
 [uBlock Origin's manifesto](MANIFESTO.md).
@@ -171,7 +66,3 @@ to maintain the filter lists you are usi
 all for free.
 
 You can contribute by helping translate uBlock₀ [on Crowdin](https://crowdin.net/project/ublock).
-
-## License
-
-[GPLv3](https://github.com/gorhill/uBlock/blob/master/LICENSE.txt).
